cmd/infra/aws/create_iam.go

```diff
@@ -63,12 +63,16 @@ func NewCreateIAMCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.AWSCredentialsFile, "aws-creds", opts.AWSCredentialsFile, "Path to an AWS credentials file (required)")
 	cmd.Flags().StringVar(&opts.InfraID, "infra-id", opts.InfraID, "Infrastructure ID to use for AWS resources.")
+	cmd.Flags().StringVar(&opts.OIDCBucketName, "oidc-bucket-name", "", "The name of the bucket in which the OIDC discovery document is stored")
```

Comment:
can we be opinionated about this using the infra-id?

Reply:
nevermind, we are sharing the bucket among HCs.  before, we had a bucket per HC and thus could just generate the bucket name and have an opinionated path within the bucket, thus nothing was required from the user.